pyproject.toml

```diff
@@ -6,11 +6,8 @@ build-backend = "hatchling.build"
 source = "vcs"
 
 [tool.hatch.build.targets.wheel]
-packages = ["bids/ext/reports"]
-
-[tool.hatch.build.targets.wheel.sources]
-"src" = ""
-"scripts" = "mypackage/scripts"
+packages = ["bids"]
+only-include = ["bids/ext/reports"]
```

Comment:
For your reference, this was the main issue: https://hatch.pypa.io/latest/config/build/#explicit-selection

Everything else just followed from fixing tests